src/main/webapp/app/entities/flair-bi/tableview/flair-bi-table-view.controller.js

```diff
@@ -0,0 +1,100 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('flairbiApp')
+        .controller('FlairBiTableviewController', FlairBiTableviewController);
```

Comment:
still you did not rename the controller. file name or controller should be self expaintanoty related to functionality